contentcuration/contentcuration/frontend/channelList/views/Channel/CatalogFilters.vue

```diff
@@ -75,11 +75,13 @@
         <div class="subheading">
           {{ $tr('includesLabel') }}
         </div>
-        <Checkbox v-model="coach" color="primary">
-          <template #label>
```

Comment:
Hi @BabyElias! it seems like `#label` should not be here and is causing the problem? Just to be consistent with our practice I think this code should still live in `<template>` with a `<span>` and `<HelpTooltip>`. I hope removing the label works! Thanks

Reply:
Also I'm not sure if after the refactor @MisRob has mentioned, do we still need to follow this pattern?

Reply:
Removing the #label does work. Thanks for your insights! Made the required changes.

Reply:
> Also I'm not sure if after the refactor @MisRob has mentioned, do we still need to follow this pattern?

@ozer550 I am not aware of any need for it. What I've seen in Vue generally, `<template>` is typically used for named slots, conditional rendering or looping groups of elements wrapped in `<template>`. 

I don't think that either option will cause any issues, so feel free to go ahead with any of them as far as I'm concerned.

Reply:
Hi @BabyElias! Now that we know the template pattern is not necessary. we can look the difference between two from UI's perspective, and it seems like the prior one that you implemented seems a bit better due to proper gap between helpToolTip and the text! Apologies, we had to re-iterate over this.

Reply:
No issues! The UI perspective does sound more reasonable. Reverted the previous changes.